src/tests/ftest/launch.py

```diff
@@ -1414,14 +1419,15 @@ def install_debuginfos():
     # installation
     path = os.path.sep + os.path.join('usr', 'share', 'spdk', 'include')
     if os.path.islink(path):
-        cmds.append(["sudo", "rm", "-f", path])
+        cmd_list.append(["sudo", "rm", "-f", path])
 
     if USE_DEBUGINFO_INSTALL:
         yum_args = [
             "--exclude", "ompi-debuginfo", "libpmemobj", "python", "openmpi3"]
-        cmds.append(["sudo", "yum", "-y", "install"] + yum_args)
-        cmds.append(["sudo", "debuginfo-install", "--enablerepo=*-debuginfo",
-                     "-y"] + yum_args + ["daos-server", "gcc"])
+        cmd_list.append(["sudo", "dnf", "debuginfo-install", "-y"] + yum_args)
+        cmd_list.append(
+            ["sudo", "dnf", "debuginfo-install", "-y"] +
+            yum_args + ["daos-server", "gcc"])
```

Comment:
It seems that these are both now `dnf debuginfo-install` commands and installing the same `yum_args` list.  Probably these can be collapsed down into a single command:
```suggestion
        cmd_list.append(["sudo", "dnf", "debuginfo-install", "-y"] + yum_args + 
                                   ["daos-server", "gcc"])
```
It looks ultimately in fact like the first command is actually superfluous in that the second one is a super-set of it.